.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -44,7 +44,7 @@ body:
       - This bug is specific to multiplayer
       - This bug is a graphical glitch or error
       - This issue has to do with park objects and/or asset packs
-      - Issues building the game
```

Comment:
I wonder whether people will also not confuse "development" with the "develop" branch. Would it be better off just adding a code at the end?
"Issues building the game code"

Reply:
Yeah, I considered that, but not all development issues are related to the code. For example, when dependencies are not fetched successfully or something.

Reply:
Maybe "issues compiling the code" or something like that?